runtimes/eoapi/stac/eoapi/stac/templates/items.html

```diff
@@ -8,125 +8,128 @@
   {% set urlq = url + '?' %}
 {% endif %}
 
-
+{% block content %}
 <nav aria-label="breadcrumb">
   <ol class="breadcrumb bg-light">
     {% for crumb in crumbs %}
       {% if not loop.last %}
-    <li class="breadcrumb-item"><a href="{{ crumb.url }}/">{{ crumb.part }}</a></li>
-      {% else %}<li class="breadcrumb-item active" aria-current="page">{{ crumb.part }}</li>
+      <li class="breadcrumb-item"><a href="{{ crumb.url }}/">{{ crumb.part }}</a></li>
+      {% else %}
+      <li class="breadcrumb-item active" aria-current="page">{{ crumb.part }}</li>
       {% endif %}
     {% endfor %}
 
     <li class="ml-auto json-link"><a target="_blank" href="{{ urlq }}f=geojson">GeoJSON</a></li>
   </ol>
 </nav>
 
-<h1>Collection Items: {{ response.title or response.id }}</h1>
-
-<div id="map" class="rounded" style="width:100%; height:400px;">Loading...</div>
-
-<p>
-  <b>Number of matching items:</b> {{ response.numberMatched }}<br/>
-  <b>Number of returned items:</b> {{ response.numberReturned }}<br/>
-</p>
```

Comment:
@oliverroick can we add those back? it's always good to know how many items are available

Reply:
note `response.numberMatched` will only be set if the `context` extension is enabled in your pgstac instance 

```
psql -d postgres -h 0.0.0.0 -p 5439 -U username -c 'SET SEARCH_PATH to pgstac, public; ALTER ROLE username SET pgstac.context TO 'on';'
```

Reply:
Note in the collections page we have a text like: `Showing 1 - 10 of 30 collections`

Reply:
Ah yes, I remove that because I didn't see the `numberMatched`. I've added `Showing 100 of 395 items`, similar to what we have on the collections page.